src/RefractDataStructure.cc

```diff
@@ -453,6 +511,7 @@ namespace drafter {
             }
         };
 
+        /*
```

Comment:
This one of parts need to be polished, another one is RefractDataStructure.cc#L1024
Additionaly all `Extract*` functors should inherited from one ancestor.